src/js/containers/covid19/assistanceListing/SpendingByCFDAContainer.jsx

```diff
@@ -347,6 +382,17 @@ const SpendingByCFDAContainer = ({ activeTab, scrollIntoView }) => {
                 resultsText
                 pageSize={pageSize}
                 totalItems={totalItems} />
+            <CFDADetailModal
```

Comment:
You could throw this inside `GlobalModalContainer` and just reuse our existing `showModal` actions to show it. Then here, instead of adding two modal components, you would just render `<GlobalModalContainer />`